tests/main/interfaces-kerberos-tickets/task.yaml

```diff
@@ -0,0 +1,46 @@
+summary: Ensure that Kerberos tickets integration works
+
+details: |
+    Snap_run overwrites the Kerberos cache via the environment.
+    Check that that is working as expected.
+
+    Also ensure that the snap can read a ticket in the cache
+    if and only if the kerberos-ticket interface is connected,
+    and can never write to it.
+
+systems: [ubuntu-24.04-64]
```

Comment:
Could we run this on a wider array of systems or did you want to limit testing time?

Reply:
I just wanted to limit testing time. Is it preferrable to remove this statement and let it run wherever?

Reply:
Yes